components/playground/instance/tidb_config_test.go

```diff
@@ -0,0 +1,33 @@
+package instance
+
+import (
+	"bytes"
+	"os"
+	"testing"
+
+	"github.com/BurntSushi/toml"
+	"github.com/pingcap/tidb/config"
```

Comment:
We are avoiding dependency upon `tidb` and other components by intend, please use general interface objects (`map[string]interface{}` with type assertion) to decode the config.

Or simply provide a command line argument in `playground` to specify TiDB port and overwrite the default.